DEVELOPMENT.md

```diff
@@ -270,11 +270,12 @@ Replace the `src/contrib/bootstrap/` directory with the contents of the
 downloaded zip file. Be sure to update this list of components after each new
 download.
 
-## Releases/Changelog
+## Releases
 
 The version in `package.json` is updated automatically with the `grunt
 release:$part` task (where `$part` is `patch`, `minor`, or `major`). This task
 creates a git tag for the new version, and adds 2 new header lines to
 `CHANGELOG.md`, indicating the version number and the date it was
-released. Therefore, new changes should be documented at the top of CHANGELOG.md
-without a version header, since it will be added automatically later.
+released. Note that no `## Unreleased` place-holder line is necessary at the top of the
+`CHANGELOG.md` file. Simply add a description of changes at the beginning of `CHANGELOG.md`
+and the release version and date will be inserted when the version is `bump`-ed.
```

Comment:
Not saying we should do this now, but I wonder if it's worth a tech debt story to have it replace the "## Unreleased" tag as the preferred way to do it, so that it's the same process as our other projects?

Reply:
I thought about this as well. I'll add it to my list of possible follow-up stories.